packages/Molecules/Form/fields/fieldRplselect.vue

```diff
@@ -1,118 +1,384 @@
-    addTag (newTag, id) {
-      let onNewTag = this.selectOptions.onNewTag
-      if (typeof onNewTag === 'function') {
-        onNewTag(newTag, id, this.options, this.value)
+    removeOutsideTest (event) {
+      if (typeof window !== 'undefined') {
+        document.removeEventListener('click', this.testOutside)
       }
     },
-    onSearchChange (searchQuery, id) {
-      let onSearch = this.selectOptions.onSearch
-      if (typeof onSearch === 'function') {
-        onSearch(searchQuery, id, this.options)
+    preventKeyboardScroll (e) {
+      // up down and space keys
+      const keys = ['Up', 'ArrowUp', 'Down', 'ArrowDown', ' ', 'Space'] 
+      if (this.isOpen && keys.includes(e.key)) {
+        e.preventDefault()
       }
     }
+  },
+  mounted () {
+    document.body.addEventListener('keydown', this.preventKeyboardScroll)
+    if (this.schema.multiselect && !this.value) {
+      this.value = []
+    }
+  },
+  beforeDestroy () {
+    document.body.removeEventListener('keydown', this.preventKeyboardScroll)
   }
 }
 </script>
+
+<style lang="scss">
+@import "~@dpc-sdp/ripple-global/scss/settings";
+@import "~@dpc-sdp/ripple-global/scss/tools";
+@import "../scss/form";
+
+$rpl-select-inner-padding: $rpl-space-4;
+$rpl-select-dropdown-height: 10rem !default;
+
+.rpl-select {
+  $root: &;
+
+  &__native {
+    position: relative;
+    @include rpl-breakpoint("s") {
+      @include rpl_visually_hidden;
+    }
+    select {
+      @include rpl_form_text_element;
+      width: 100%;
+      -webkit-appearance: none;
+      -moz-appearance: none;
+      appearance: none;
+    }
+  }
+
+  &__inner {
+    display: none;
+    @include rpl_form_text_element;
+    @include rpl-breakpoint("s") {
+      padding: 0;
+      display: block;
+    }
+  }
+
+  &__trigger {
+    padding: $rpl-form-element-padding-m-horizontal;
+    position: relative;
+
+    &-icon {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      right: 1rem;
+      margin: auto;
+      transition: transform .25s;
+
+      #{$root}--open & {
+        transform: rotate(-180deg);
+      }
+    }
+
+  }
+
+  &__dropdown {
+    top: 0;
+    background-color: $rpl-form-element-bg-color;
+    position: relative;
+    width: 100%;
+  }
+
+  &__listbox {
+    outline: 0;
+    position: absolute;
+    top: -1px;
+    z-index: 1;
+    background: $rpl-form-element-bg-color;
+    left: -1px;
+    width: 100%;
+    border: 1px solid $rpl-form-element-border-color;
+    border-radius: $rpl-form-element-border-radius;
+    border-top: 1px solid $rpl-form-element-border-color;
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+    height: $rpl-select-dropdown-height;
+    overflow-y: scroll;
+  }
+
+  &__listitem {
+    padding: $rpl-space-3 $rpl-form-element-padding-m-horizontal;
+    background: $rpl-form-element-bg-color;
+
+    &:nth-child(odd) {
+      background-color: rpl-color("white");
+    }
+
+    &:hover {
+      background-color: rpl-color("secondary");
```

Comment:
Especially with colours, these may be better served as $variables for possible theme changes.

Reply:
Have added focus and selected colours as variables